src/input.ts

```diff
@@ -27,15 +27,51 @@ export interface Input {
     contentType?: string;
 }
 
+export interface ConnectionString {
+    type: 'connection-string';
+    connectionString: string;
+}
+
+export interface Identity {

```

Comment:
I would prefer if we rename this to WorkloadIdentity, instead of Identity. ServicePrincipal is also an Identity

Reply:
Also change the type